src/pkg/bb/bb.go

```diff
@@ -200,6 +204,10 @@ func BuildBusybox(opts *Opts) (nerr error) {
 		return fmt.Errorf("failed to write main.go: %v", err)
 	}
 
+	if opts.GenerateOnly {
```

Comment:
Can you make it so that GenerateOnly can only be used if the user set GenSrcDir, check that at the beginning of the func?

Or, alternatively, if GenSrcDir is not set, can you do a `log.Printf("Generated source can be found in %s.", tmpDir)` and make sure that the `os.RemoveAll(tmpDir)` that is inside the defer above doesn't happen?

Reply:
how's this.